docs/2-controlling-nullability.md

```diff
@@ -60,7 +60,7 @@ The GraphQL type system distinguishes between nullable and non-nullable types. T
 
             [Required]
             [StringLength(200)]
-            public string? Name { get; set; }
```

Comment:
This is actually correct as nullable since it can be technically nullable when you create the instance. But the required attribute will still signal to GraphQL and EF that it will not be null once it is correctly initialized.